docs/ARCHITECTURE.md

```diff
@@ -17,6 +17,8 @@ For more details on our system security, see [SECURITY.md](SECURITY.md).
 #### Entity-relationship Diagram
 ![ERD](diagrams/erd.png)
 
+To update the ERD, install the [rails-erd gem](https://github.com/voormedia/rails-erd) (`gem install rails-erd`) and run `erd` from the project root. The ERD is generated using the `.erdconfig` file, also in the project root.
```

Comment:
the `rails-erd` gem is already in the Gemfile, so maybe just shoren to `bundle exec erd` ?